package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-native-dialog",
-  "version": "6.1.2",
+  "version": "6.1.3",
```

Comment:
Can we leave it as it is? The minor will be bumped automatically once merged 👍

Reply:
Okay, have been committed that :)